proxy-node-translator/src/main/java/uk/gov/ida/notification/translator/saml/HubResponseTranslator.java

```diff
@@ -48,19 +49,20 @@ Response getTranslatedHubResponse(HubResponseContainer hubResponseContainer) {
         final String pid = pidPrefix + hubResponseContainer.getPid();
 
         if (hubResponseContainer.getVspScenario().equals(VspScenario.IDENTITY_VERIFIED)) {
+            var attributes = getAttributesOrThrow(hubResponseContainer);
```

Comment:
I don't think we need a separate method here - fine to just inline the logic in `getAttributesOrThrow`